etcdserver/cluster_util.go

```diff
@@ -94,7 +94,16 @@ func getClusterFromRemotePeers(urls []string, timeout time.Duration, logerr bool
 			}
 			continue
 		}
-		return membership.NewClusterFromMembers("", id, membs), nil
+		// check the length of membership members
+		// if the membership members are present then prepare and return raft cluster
+		// if membership members are not present then the raft cluster formed will be
+		// an invalid empty cluster hence return an no raft cluster member error
+		if len(membs) > 0 {
+			return membership.NewClusterFromMembers("", id, membs), nil
+		} else {
+			return nil, fmt.Errorf("could not detect raft cluster member(s) from the urls.")
```

Comment:
@xiang90  you asked to remove else statement. i am confused whether you want me to remove else or correct the error message as specified by you. Let me know and i will correct it and will re-submit after doing corrections.